### PR TITLE
OSDOCS:2475: Adding Dynamic Plugins as Tech Preview in Rel Notes

### PR DESCRIPTION
**Pull request by opayne1**
[OSDOCS:2475](https://issues.redhat.com/browse/OSDOCS-2475)
[OSDOCS:2478](https://issues.redhat.com/browse/OSDOCS-2478)

Adds Dynamic Plugins and the Multicluster console to Tech preview table as well as the web console section in Rel Notes.

- Applies to `enterprise-4.10` only
- [Preview link](https://deploy-preview-40759--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview)

### CODE REVIEW

**Comment by opayne1** [comment]
@spadgett @kdoberst PTAL. Thank you!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -132,6 +132,14 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-dynamic-plugin-technology-preview"]
+==== Dynamic Plug-in (Technology Preview)
+Starting with {product-title} 4.10, OpenShift console dynamic plug-ins are now available as a Technology Preview feature. This feature allows you make customizations to your interface at runtime such as: add custom pages, add perspectives and update navigation items, and add tabs and actions to resource pages. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
+
+[id="ocp-4-10-multicluster-console-technology-preview"]
+==== Multi-cluster Console (Technology Preview)
+Starting with {product-title} 4.10, the multi-cluster console is now available as a Technology Preview feature. This feature supports a single {product-title} console that can connect to remote clusters API. An *All clusters* perspective contains the Advanced Cluster Management(ACM) multi-cluster views. This feature also implements single sign-on(SSO), so you can bypass the login prompt when navigating between clusters.
```
Comment (spadgett) [inline release_notes/ocp-4-10-release-notes.adoc:141]: We'll need to document how to enable the multi-cluster feature somewhere. It requires setting a feature gate.

The "All clusters" perspective will only be available if you have ACM 2.5 installed and enable the ACM console plugin, which is a manual step an admin needs to take.

I'd remove or rephrase the sentence about SSO. We have an IDP config operator that will push configuration for either GitHub or LDAP identity providers to all clusters in the fleet, but it's a separate operator to install and configure.

cc @alimobrem @showeimer @jamestalton @robinbobbitt

**Review by spadgett — APPROVED** [review]
lgtm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -132,6 +132,14 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-dynamic-plugin-technology-preview"]
+==== Dynamic Plug-in (Technology Preview)
+Starting with {product-title} 4.10, OpenShift console dynamic plug-ins are now available as a Technology Preview feature. This feature allows you make customizations to your interface at runtime such as: add custom pages, add perspectives and update navigation items, and add tabs and actions to resource pages. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
```
[the remark this answers is not in the document]
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:140]: See #41260 for the docs.

**Comment by opayne1** [comment]
@yapei Could I go ahead and get your ack for these notes? I hope to have the PRs for these links to you this week. Thank you!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -1335,7 +1345,17 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
-|Pod-level bonding for secondary networks
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1338]: I think something happened in rebase here.

**Comment by yapei** [comment]
except for the link we are going to add, others look good to me

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -177,6 +177,16 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-dynamic-plugin-technology-preview"]
+==== Dynamic plug-in (Technology Preview)
+Starting with {product-title} 4.10, OpenShift console dynamic plug-ins are now available as a Technology Preview feature. This feature allows you make customizations to your interface at runtime such as: add custom pages, add perspectives and update navigation items, and add tabs and actions to resource pages. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-10-release-notes.adoc:182]: I might add a list here and try to make the first part more user-focused. (Saying that the program allows you to do something is a bit anthropomorphic.) Maybe something like:

```suggestion
Starting with {product-title} {product-version}, the ability to create OpenShift console dynamic plug-ins is now available as a Technology Preview feature. You can use this feature to customize your interface at runtime in many ways, including:

* Adding custom pages
* Adding perspectives and updating navigation items
* Adding tabs and actions to resource pages

After you register a plug-in with the console by using the `ConsolePlugin` custom resource, a cluster administrator must enable them in the console Operator configuration.
```

Are you creating multiple plug-ins? (I want to make sure we're describing the single feature correctly. It sounds like the feature is the ability to make a plug-in.)
Can anyone register the plug-in?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -177,6 +177,16 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+For more information on the dynamic plug-in, see xref:../web_console/dynamic-plug-ins.adoc#dynamic-plug-ins_dynamic-plug-ins[Adding a dynamic plug-in to the {product-title} web console].
+
+[id="ocp-4-10-multicluster-console-technology-preview"]
+==== Multi-cluster console (Technology Preview)
+Starting with {product-title} 4.10, the multi-cluster console is now available as a Technology Preview feature. This feature supports a single {product-title} console that can connect to remote clusters API.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-10-release-notes.adoc:188]: Maybe:

```suggestion
Starting with {product-title} 4.10, the multi-cluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console.
```

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of questions and suggestions, but this is sounding good!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 5 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:188]: Added suggestion.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 4 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:182]: You actually don't need admin privileges. Updating this sentence.

Comment (spadgett) [reply release_notes/ocp-4-10-release-notes.adoc]: You need to have authority to create a `ClusterPlugin` resource. By default, only cluster admins can create one.

**Pull request merged**